app/components/RequestCard.tsx

```diff
@@ -45,15 +72,16 @@ export const RequestCard = (props: RequestCardProps) => {
               paddingBlock="$2"
               paddingInline="$3"
               borderRadius="$radius.8"
+              style={$requestTitleStyle}
```

Comment:
nonblocking but interesting that typescript doesn't like this, when I checked out your branch and ran tsc checks (`npm run compile`) it was chill
 
```suggestion
maxWidth={140}
alignSelf="flex-start"
flexShrink={0}
```

Reply:
I will give this a shot again, pre-commits wasn't letting me but I can `-n` it